mathml/relations/css-styling/dynamic-dir-1.html

```diff
@@ -1,27 +1,31 @@
 <!DOCTYPE html>
-<html>
+<html class="reftest-wait">
 <head>
   <meta charset="utf-8">
   <title>Test dynamically changing dir attribute</title></head>
   <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#cssproperties">
+  <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#dom">
   <meta name="assert" content="The dir attribute should update direction map to css properties dynamically">
   <link rel="match" href="dynamic-dir-1-ref.html">
   <script>
-      window.addEventListener("DOMContentLoaded", function() {
-        var x = document.getElementsByTagName("math");
-        x[0].setAttribute("dir", "rtl");
-        x[1].setAttribute("dir", "rtl");
-        x[2].removeAttribute("dir");
+      window.addEventListener("DOMContentLoaded", () => {
+        ["math", "mstyle", "mrow"].forEach((tag) => {
+          let elements = document.getElementsByTagName(tag)
+          
+          // set an explcit rtl where there was none
+          elements[0].setAttribute("dir", "rtl");
 
-        x = document.getElementsByTagName("mstyle");
-        x[0].setAttribute("dir", "rtl");
-        x[1].setAttribute("dir", "rtl");
-        x[2].removeAttribute("dir");
-    
-        x = document.getElementsByTagName("mrow");
-        x[0].setAttribute("dir", "rtl");
-        x[1].setAttribute("dir", "rtl");
-        x[2].removeAttribute("dir");
+          // change explcit ltr to rtl
+          elements[1].setAttribute("dir", "rtl");
+
+          // remove an explicitly set dir="rtl"
+          elements[2].removeAttribute("dir");
+
+        })
+
```

Comment:
Sorry I was not clear. The idea is that after DOMContentLoaded the MathML might not have been laid out so the test is not really checking dynamic layout change. That's why the original test used MozReftestInvalidate. I think @emilio suggestion is basically:

window.addEventListener("load", () => {
  document.documentElement.getBoundingClientRect(); // force layout
  ["math", "mstyle", "mrow"].forEach((tag) => {
     ...
  });
  document.documentElement.classList.remove('reftest-wait');
});

See https://gist.github.com/paulirish/5d52fb081b3570c81e3a

The original test didn't force layout again before reftest-wait removal, maybe we want to do that though.

Reply:
Ah! I get it now, thanks for clarifying that -- I wasn't perceiving that as forcing _initial_ layout and chose not to do that before reftest-wait removal because I thought that part of the test was to be sure that invalidation was happening.  I'll update the thing that starts it as you say, but that second still seems right to me, we don't want to force it before removing, right?

Reply:
Right, I guess we should avoid forcing the second layout as the test is really about checking it happens anyway (my concern is whether it will happen before the screenshot, which does not seem guaranteed by the reftest-wait stuff).